Leaflet.Graticule.js

```diff
@@ -13,6 +13,8 @@ L.LatLngGraticule = L.Layer.extend({
         font: '12px Verdana',
         lngLineCurved: 0,
         latLineCurved: 0,
+        dmd: false, // Labels: Display Degrees - Decimal Minutes instead of Decimal Degrees
```

Comment:
I would like a `format` options instead (with `dmd` as possible value) instead. There are more than 2 coordinate formats.

Reply:
I'll put an option named `format` which can have one of the following values:
- **dd**: decimal degrees (default value)
- **dmd**: degrees - Decimal Minutes
- **dms**: degrees - Minutes - Second